library/core/src/result.rs

```diff
@@ -1626,6 +1626,25 @@ impl<A, E, V: FromIterator<A>> FromIterator<Result<A, E>> for Result<V, E> {
     }
 }
 
+//FIXME: Figure out how to mark this as unstable without x.py refusing to run any tests
+#[stable(feature = "tuple_from_result_iter", since = "1.53.0")]
```

Comment:
This obviously should be marked as unstable, but every time I did so, it would cause `x.py` to refuse any attempt I made to run the tests, instead exiting with the message:

```
error: an `#[unstable]` annotation here has no effect
    --> library/core/src/result.rs:1630:1
     |
1630 | #[unstable(feature = "tuple_from_result_iter", issue = "87047")]
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: `#[deny(ineffective_unstable_trait_impl)]` on by default
     = note: see issue #55436 <https://github.com/rust-lang/rust/issues/55436> for more information
```

I realize there is probably just a mistake I'm making as a result of this being my first contribution here, so I'd appreciate some help resolving this small detail.

Reply:
Impls can't be unstable, see the linked issue. They're always insta-stable.

Reply:
I see. I read the issue before but was confused because there is an `impl` just a few lines below where I inserted my code (line 1649, `impl<T, E> ops::TryV2 for Result<T, E>`) which _is_ marked as unstable, but upon re-reading the issue I realize the difference is that there the trait itself is unstable.